### ISSUE TITLE
Cannot send + in the request

### ISSUE THREAD

**Issue by roly445** [TrevorPilley/Net.Http.WebApi.OData#78]
Hi,

I just updated to 3.3.5 yesterday and cannot send a '+' in the url.  I see that this is because of #58.

I try to urlEncode the variable first, but with luck. The Url I'm trying to send up is base64 encoded variable so its going to have more than one + in it.

I understand why we need to swap + with a space, but could we set that up with some kind of configuration?

**Comment by TrevorPilley** [comment]
Yeah sure, can you give me an example URL or better yet a failing test to work from?

**Comment by TrevorPilley** [comment]
@roly445 I should have time to look at this over the weekend if you can give me an example URL or failing test to work from - also open to a PR with a proposed fix if you'd like to tackle it yourself?

**Comment by TrevorPilley** [comment]
Any updates @roly445?

**Comment by roly445** [comment]
Sorry @TrevorPilley I have been away from my computer from the last few weeks.

I'll put something together now

**Comment by roly445** [comment]
#79
